src/app/check/page.tsx

```diff
@@ -1,7 +1,14 @@
-import React from 'react'
+
+'use client'
```

Comment:
better clean up the check page after testing. if someone's also working in this part, and push, we'll get merge conflict.